src/gt4py/next/program_processors/runners/dace_iterator/itir_to_tasklet.py

```diff
@@ -388,15 +411,15 @@ def _add_symbol(self, param, arg):
             # create storage in lambda sdfg
             ndims = len(arg.dimensions)
             shape = tuple(
-                dace.symbol(unique_var_name() + "__shp", dace.int64) for _ in range(ndims)
+                dace.symbol(unique_var_name() + "__shp", _SYMBOL_DTYPE) for _ in range(ndims)
```

Comment:
You should switch to `unique_name(...)` further you should include the name and which stride it is meaning:
```
dace.symbol(unique_name(f"__{param}_shp{i}"), _SYMBOL_DTYPE) for i in range(ndims) 
```

Reply:
I agree. I will create a utility function to do this, it could also be used in #1379